package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "vaadin",
   "displayName": "vaadin",
-  "description": "Official Vaadin plugin for VSCode, currently featuring undo support for Vaadin Copilot modifications.",
-  "version": "0.0.1",
+  "description": "Official Vaadin plugin for VS Code. Create Vaadin projects and improve Vaadin Copilot experience!",
+  "version": "1.0.0",
```

Comment:
version here does not matter as it is replaced on Bender. It's only for generated packages to have nicer name.